athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/NeptuneRecordHandler.java

```diff
@@ -79,13 +63,13 @@
     public NeptuneRecordHandler() 
     {
         this(AmazonS3ClientBuilder.defaultClient(), AWSSecretsManagerClientBuilder.defaultClient(),
-                AmazonAthenaClientBuilder.defaultClient(),
-                new NeptuneConnection(System.getenv("neptune_endpoint"), System.getenv("neptune_port"), Boolean.parseBoolean(System.getenv("iam_enabled"))));
+                AmazonAthenaClientBuilder.defaultClient(), new NeptuneConnection(System.getenv("neptune_endpoint"),
+                        System.getenv("neptune_port"), Boolean.parseBoolean(System.getenv("iam_enabled"))));
     }
 
     @VisibleForTesting
-    protected NeptuneRecordHandler(final AmazonS3 amazonS3, final AWSSecretsManager secretsManager,
-            final AmazonAthena amazonAthena, final NeptuneConnection neptuneConnection) 
+    public NeptuneRecordHandler(final AmazonS3 amazonS3, final AWSSecretsManager secretsManager,
```

Comment:
Why are you changing the visibility of this method? it doesn't seem like it needs to be public, especially since you left it marked as @VisibleForTesting

Reply:
reverted